src/components/jobs_view.tsx

```diff
@@ -128,25 +156,33 @@ const JobList: React.FC<JobListProps> = ({ dbName }) => {
     const handleAddJob = async (input: string) => {
         // adding a new job here
         const docName = input
-        if (name !== null) {
-            await putNewDoc(db, docName)
+        if (docName !== null) {
+            //await putNewDoc(db, docName)
```

Comment:
Here's another commented out line without an explanation.

Reply:
Removed the comment code that is not needed.